### PR TITLE
Copy env message locally; fix minor compilation warnings

### PR DESCRIPTION
**Pull request by mattcolegate**

### CODE REVIEW

File: Sources/SwiftMetrics/SwiftMonitor.swift [hunk 1]
```diff
@@ -183,6 +183,7 @@ public class SwiftMonitor {
           number.of.processors=2
           command.line=/home/exampleuser/SwiftMetrics/sample/.build/debug/test
           */
+      print("Incoming message: \(message)")
```
Comment (tobespc) [inline Sources/SwiftMetrics/SwiftMonitor.swift]: Should this print be left in ?

**Comment by tfrank64** [comment]
@mattcolegate @tobespc Looks like that print statement should be removed, was anyone planning on doing this soon? I've been seeing these compilation warnings too.

File: Sources/SwiftMetrics/SwiftMonitor.swift (hunk 1 above)
Comment (mattcolegate) [reply Sources/SwiftMetrics/SwiftMonitor.swift]: No - I'll remove it

**Comment by mattcolegate** [comment]
@tobespc can we merge please?

**Comment by tobespc** [comment]
looks like the builds are failing due to an error in SwiftyJSON ?.   I'm going to accept the change request regardless as this is holding up other testing

**Pull request merged**